Trace/Helper/Swift/Double+Helper.swift

```diff
@@ -36,31 +36,22 @@ extension Double {
     
     /// 123.45 split to 123 and 45
```

Comment:
Would the comment need to change to
`/// 123.45 split to 123 and 450000000`?

Reply:
fixed

Reply:
:+1: